adapters/sinatra/lib/opentelemetry/adapters/sinatra/extensions/tracer_extension.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require_relative '../middlewares/tracer_middleware'
+
+module OpenTelemetry
+  module Adapters
+    module Sinatra
+      module Extensions
+        module TracerExtension
+          # Sinatra hook after extension is registered
+          def self.registered(app)
+            # Create tracing `render` method
+            ::Sinatra::Base.module_eval do
+              def render(engine, data, *)
+                Sinatra::Adapter.tracer.in_span(
+                  'sinatra.render_template',
+                  kind: :server,
+                  with_parent: Sinatra::Adapter.tracer.current_span
```

Comment:
This shouldn't be necessary. The `current_span` will be used as the parent by default.

Reply:
I've removed `with_parent:` :heavy_check_mark: